schedule/yast/maintenance/autoyast_minimal_no_updates.yaml

```diff
@@ -0,0 +1,20 @@
+name: autoyast_minimal_no_updates
+description:    >
+  Test performs autoyast installation to generate minimal qcow image without self updates.
```

Comment:
We need to be more specific here what kind of system we are installing, it is not really clear, in my understanding of the bug (but I'm might be wrong, needs to be tested) we need a system which is minimal (that we could be done with system role or in other ways) and registered (so we can do zypper ar and dup later) but must not have update repositories (so when you do the zypper dup only will pick the repo for self update). At the same time the system cannot be installed with self_update (otherwise why we would be adding this repo later).